### DISCUSSION TITLE
Making a MolBlock from Conformer

### DISCUSSION THREAD

**Discussion by testst2002** [rdkit/rdkit#4301] [Q&A]
I need to make sdf file with molecule before and after changing coords. I changed coords by using conformer, but I don't know how can I make MolBlock form Conformer. Is there any ways to do it? 
Or mb I can write it somehow else?
If it'll help here is how I change coords:
![image](https://user-images.githubusercontent.com/86313385/124114278-0f709f80-da75-11eb-879f-990387ee04a5.png)

**Comment by greglandrum** [comment]
@testst2002 this looks like it could work. What are you observing which makes you think that it isn't?
